### PR TITLE
Update notice.txt file and format the cgmanifest.json file.

### PR DESCRIPTION
**Pull request by ahsonkhan**
Fixes https://github.com/Azure/azure-sdk-for-c/issues/1329

### CODE REVIEW

**Comment by vhvb1989** [comment]
How do we know if a ticket hasn't created before to review the dependencies here?
Shouldn't we wait until any previous review is done?

File: cgmanifest.json [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-                    "RepositoryUrl": "https://github.com/xbmc/libssh",
-                    "CommitHash": "3cb0a1bf023cce7209290f998b8dc3029ed3df77"
-                }
-            },
-            "DevelopmentDependency": false
+  "Registrations": [
+    {
+      "Component": {
+        "Type": "git",
+        "git": {
```
Comment (ahsonkhan) [inline cgmanifest.json:6]: This was the issue. It shouldn't be `"Other"`, it should be `"git"`.

**Review by vhvb1989 — APPROVED** [review]
👏

File: cgmanifest.json [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-    "Registrations": [
-        {
-            "Component": {
-                "Type": "git",
-                "Other": {
-                    "RepositoryUrl": "https://github.com/bilke/cmake-modules",
-                    "CommitHash": "66fa2579d36f30cdab93d1dbd4c3d682a25ae5f3"
-                }
-            },
-            "DevelopmentDependency": false
```
Comment (ahsonkhan) [inline cgmanifest.json:11]: I changed `DevelopmentDependency` to true. From what I understand, false is the default and is meant for "true" dependencies. Our dependencies here are for code coverage cmake module and dev-time script, and it doesn't impact the SDK product directly.

**Pull request merged**